packages/odyssey-react-mui/src/labs/UiShell/UiShellContent.tsx

```diff
@@ -25,16 +25,11 @@ import { useScrollState } from "./useScrollState";
 
 const emptySideNavItems = [] satisfies SideNavProps["sideNavItems"];
 
-const StyledAppContainer = styled("div", {
-  shouldForwardProp: (prop) => prop !== "odysseyDesignTokens",
-})<{
-  odysseyDesignTokens: DesignTokens;
-}>(({ odysseyDesignTokens }) => ({
+const StyledAppContainer = styled("div")(() => ({
   gridArea: "app-content",
   overflowX: "hidden",
   overflowY: "auto",
-  paddingBlock: odysseyDesignTokens.Spacing5,
-  paddingInline: odysseyDesignTokens.Spacing8,
+  backgroundColor: "white",
```

Comment:
nit: you could alternatively use `odsseyDesignTokens.HueNeutralWhite` here

Reply:
@benlister-okta Updated